kube-config-files/secrets-templates/binary-writer-template.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: binary-writer
```

Comment:
I just realized that I need also these keys, so I think we should name them more generically, like aws-secrets